src/main/java/book/store/model/Order.java

```diff
@@ -0,0 +1,58 @@
+package book.store.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Getter
+@Setter
+@EqualsAndHashCode(of = {"id"})
+@Entity
+@SQLDelete(sql = "UPDATE orders SET is_deleted = TRUE WHERE id=?")
+@SQLRestriction(value = "is_deleted = FALSE")
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false)
+    @MapsId
+    private User user;
+    @Column(nullable = false)
+    private Status status;
+    @Column(nullable = false)
+    private BigDecimal total;
+    @Column(nullable = false)
+    private LocalDateTime orderDate;
```

Comment:
```suggestion
    private LocalDateTime orderDate = LocalDateTime.now();
```
You can initialise orderDate here